simulator/infra/ports/module.cpp

```diff
@@ -48,6 +48,16 @@ void Module::enable_logging_impl( const std::unordered_set<std::string>& names)
         c->enable_logging_impl( names);
 }
 
+// enable-prefetch function implementation
+void Module::enable_prefetch(bool key, uint32 fetchahead_size, std::string prefetch_method)
+{
+    prefetch_enabled = key;
+    this->fetchahead_size = fetchahead_size;
+    this->prefetch_method = prefetch_method;
+    for ( const auto& c : children)
+        c->enable_prefetch(key, fetchahead_size, prefetch_method);
+}
```

Comment:
Now we keep `config` variables locally to modules.
Although I agree there are architectural disadvantages of this, I propose to be consistent.